app/assets/javascripts/admin/controllers/admin-user-badges.js.es6

```diff
@@ -70,9 +70,20 @@ export default Ember.Controller.extend(GrantBadgeController, {
             }
           });
         },
-        function() {
-          // Failure
-          bootbox.alert(I18n.t("generic_error"));
+        function(error) {
```

Comment:
wouldn't our `popupAjaxError` helper do all this logic?

Reply:
I didn't know about this function. That function works here perfectly though had to fix a bug in it :slightly_smiling_face: